config/form-content/mandaat/form.ttl

```diff
@@ -16,14 +16,6 @@ ext:aantalHoudersF
     sh:name "Aantal houders";
     sh:order 2;
     sh:path mandaat:aantalHouders.
-ext:bestuursfunctieCodeF
```

Comment:
don't you need to remove this from the form:includes as well?

Reply:
absolutely!